packages/astroport/src/factory.rs

```diff
@@ -50,6 +50,8 @@ pub struct PairConfig {
     /// We disable pair configs instead of removing them. If it is disabled, new pairs cannot be
     /// created, but existing ones can still obtain proper settings, such as fee amounts
     pub is_disabled: Option<bool>,
+    /// Is the generator of the pair disabled
```

Comment:
/// Setting to true will disable creating a pool in generator for this pair type

Reply:
You're right, it's better. Changed